tests/e2e/test_all.yml

```diff
@@ -0,0 +1,20 @@
+- name: Migration tests
+  hosts: migrator
+  tasks:
+    - import_tasks: prep.yml
+    - include_tasks: clean.yml
+      args:
+        apply:
+          tags: test_clean_before
+      tags: always
+    - import_tasks: seed.yml
+    - import_tasks: run.yml
+    # TODO: Remove comment after
```

Comment:
Just tagging this to not forget about the TODO.

Reply:
done